src/docs/reference/modules/AuditTrail/README.md

```diff
@@ -0,0 +1,58 @@
+
+- a given date range,
+- categories (e.g. *All categories*, *Content*, *User*).
+
+You can also sort entries by various parameters.
+
+Events provide various details on their line:
+
+- The category and type of event (e.g. *Content* and *Published*).
+- The time when the event was recorded.
+- The user initiating the event.
+- An event-specific quick summary, if any. E.g. events of the *Content* category show you the version and title of the content item. If you click on the *Version X* link you can see the read-only editor of the content item filled with the values that the content item has at that version. If you click on the display text of the content item, you can edit the latest version of the content item.
+- Event-specific buttons, if any. E.g. events of the *Content* category will display a *View* button that you can use to view the content item at the given version, and a *Restore* button that you can use to restore the content item to the version even if the item was deleted (the restored item will be created as a draft version that you then need to publish).
+- A *Details* link. Here you can get more general information about the given event, as well as event handlers can provide custom information. E.g. if you check out the Detail view of a *Content* event you can also see the textual differences between the current version of the content item and the previous version under the *Diff* tab. The values of the current version will be shown in green, and the values of the previous version will be shown in red.
+
+## Audit Trail Settings 
+
+If you navigate to Configuration -> Settings -> Audit Trail, you will see various configuration options, depending on the enabled Audit Trail features:
+
+- A list that contains every event that you can record using the Audit Trail module. Here you can enable or disable the recording of the given events.
+- You can enable client IP address logging. When you enable this, the client IP address will be recorded in Audit Trail events. Note that depending on the legislation your site operates in, you need to take special care to collect and store such Personal Identifiable Information.
+- The *Trimming* settings are about configuring how long you would like to keep the Audit Trail events in the database. You can disable trimming if you would like to keep the events indefinitely.
+- Further tabs can be added. E.g. the *Content* tab allows you to select items of which content types you want to record Audit Trail events for.
+
+## Audit Trail Part
+
+You can attach the `AuditTrailPart` content part to your content type. This will allow content editors to enter a comment to be saved into the Audit Trail event when saving a content item. This will then be visible in the Audit Trail event list.
+
+## Recording Custom Events
```

Comment:
I've removed the long tutorial that was previously added to the file under https://github.com/OrchardCMS/OrchardCore/issues/10097. It would need extensive update to reflect the current API and while it would be nice to have such a tutorial, it's also a pain to maintain, and in the light of living examples being available in the source code I don't see the real point.